driver/queries.h

```diff
@@ -142,12 +142,15 @@ SQLRETURN EsSQLRowCount(_In_ SQLHSTMT StatementHandle, _Out_ SQLLEN *RowCount);
 #define REQ_KEY_MULTIVAL		"field_multi_value_leniency"
 #define REQ_KEY_IDX_FROZEN		"index_include_frozen"
 #define REQ_KEY_TIMEZONE		"time_zone"
+#define REQ_KEY_CATALOG			"catalog"
 #define REQ_KEY_BINARY_FMT		"binary_format"
+
+#define REST_REQ_KEY_COUNT		13 /* "query" / "cursor" count as one */
```

Comment:
The count bump was skipped in #215/#216, but fortunately to no bad outcome, since values between 9 and 15 would be treated the same by [cbor_nn_hdr_len()](https://github.com/elastic/elasticsearch-sql-odbc/blob/7becfd96b672a98bf7339af15993ec26219cf8ae/driver/queries.c#L3013).